src/Tools/CI/build.cmd

```diff
@@ -11,6 +11,9 @@ dotnet build ../../DotVVM.Framework
 dotnet restore ../../DotVVM.Framework.Hosting.AspNetCore
 dotnet build ../../DotVVM.Framework.Hosting.AspNetCore
 
+dotnet restore ../../DotVVM.Framework.Testing
```

Comment:
This file actually is not needed. We should remove it.

Reply:
OK, won't do that in this PR though. Could you please do that when you have a bit of time?